apps/events/tables.py

```diff
@@ -0,0 +1,77 @@
+import django_tables2 as tables
+from django.conf import settings
+from django.urls import reverse
+from django.utils.html import format_html
+
+from apps.events.models import EventActionType, StaticTriggerType
+from apps.utils.time import seconds_to_human
+
+
+class ActionsColumn(tables.Column):
+    def render(self, value, record):
+        trigger_type = "timeout" if record["type"] == "__timeout__" else "static"
+        view_log_url = reverse(
+            f"experiments:events:{trigger_type}_logs_view",
+            kwargs={
+                "trigger_id": record["id"],
+                "experiment_id": record["experiment_id"],
+                "team_slug": record["team_slug"],
+            },
+        )
+        edit_url = reverse(
+            f"experiments:events:{trigger_type}_event_edit",
+            kwargs={
+                "trigger_id": record["id"],
+                "experiment_id": record["experiment_id"],
+                "team_slug": record["team_slug"],
+            },
+        )
+        delete_url = reverse(
+            f"experiments:events:{trigger_type}_event_delete",
+            kwargs={
+                "trigger_id": record["id"],
+                "experiment_id": record["experiment_id"],
+                "team_slug": record["team_slug"],
+            },
+        )
+        return format_html(
+            """
+            <a class="btn btn-sm btn-outline btn-primary" href="{}">View Logs</a>
+            <a class="btn btn-sm btn-outline btn-primary" href="{}">Edit</a>
+            <a class="btn btn-sm btn-outline btn-primary" href="{}">Delete</a>""",
+            view_log_url,
+            edit_url,
+            delete_url,
+        )
```

Comment:
It would be better to have this HTML in a template file so that it get's picked up by the tailwind post-processor.

I think you could just do `get_template("template name").render(context)`